project/maintainers.md

```diff
@@ -0,0 +1,221 @@
+
+Attempts to pass this requirement by disabling tests or modifying them such that
+they are effectively disabled are strongly discouraged.  They violate the review
+goals by reducing maintainability (no new failures will be detected) and
+possibly functionality (for scenarios outside of the current mindset).
+
+At their leisure, maintainers may suggest code changes make the test suite pass.
+Doing so is never required, nor part of the minimum duties of a maintainer.
+
+### Reviewer Count
+
+Two passing reviews are required for code to be merged.  
+
+Whenever possible, the number of reviewers should be limited, as each additional
+reviewer presents an extra set of communication channels between the review and
+themselves.  One re
+
+- One reviewer has one channel between the committer and the reviewer
+- Increasing to two, adds one channel to the the committer, and one to the 
+  reviewers.
+- Increasing to three, adds one channel to the committer, and two to the
+  reviewers.
+- Increasing to four, adds one channel to the committer, and three to the
+  reviewers.
+
+The amount of possible communication grows such that 
+`commChannels(reviewers) = reviewers + reviewers(reviewers-1)/2` leads to an 
+`O(n^2)` number of channels.  Thus, keeping reviewer count low is critical to
+velocity.
+
+### Reviewer Consensus
+
+Reviewers should coordinate among themselves when differences of opinion arise
+in a review.  The first reviewer is likely to make a statement before being
+aware of the difference of opinion; but, once a difference of opinion is known,
+the reviewers should coordinate privately to find a unified presentation of the
+desired features to communicate back to the committer.
+
+The committer has no role in the evaluation of options to determine the proper
+path forward, including them only diminishes the efficiency of the process and
+increases the stress they endure while they observe the discussion.  Once a path
+is agreed upon:
+
+- If the request to the committer was reversed, the reviewer making that stance
+  should present the new path.
+- If the request to the committer was refined, the second reviewer should 
+  present the refined path.
+
+If no path forward can be agreed upon, the proposed path that is closest to the
+committer submission is the accepted path.  This guideline exists to promote
+cooperation among reviewers.  Ideas of merit which don't become part of the
+merge request should be submitted as new issues and reviewed independently.
+
+### Review Automation
+
+Reviewers have a commitment to continuous improvement of the review process.
+Whenever a reviewer sees an opportunity to reduce the manpower involved in the
+review process by automating a portion of the process, the reviewer should
+create an issue and submit it as an improvement to the CI process.
+
+Improvements are subject to the same review process and the same review goals
+as other committer offerings.  Readability, search-ability, understanding,
```

Comment:
```suggestion
as other committer offerings. Readability, search-ability, understanding,
```

Reply:
Please don't tie up review time with low quality suggestions like this in the future.